onnxmltools/convert/sklearn/operator_converters/CalibratedClassifierCV.py

```diff
@@ -0,0 +1,169 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+from ....proto import onnx_proto
+from ...common._apply_operation import apply_add, apply_cast, apply_div, apply_exp, apply_mul, apply_reshape
+from ...common._registration import get_converter, register_converter
+from .._parse import sklearn_operator_name_map
+import numpy as np
+from sklearn.naive_bayes import BernoulliNB
+from sklearn.naive_bayes import MultinomialNB
+
+
+class Oper:
```

Comment:
Humm. Is it really needed? It's a subset of `Operator` and I doubt if it will be used after we move the parsing logics into `_parse.py`.